jetty-server/src/main/config/modules/requestlog.mod

```diff
@@ -1,12 +1,39 @@
 DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
 
 [description]
-Enables the default custom request log.
+Log requests using CustomRequestLog and AsyncRequestLogWriter
 
 [tags]
 requestlog
+logging
 
-[depends]
-customrequestlog
+[depend]
+server
 
+[xml]
+etc/jetty-requestlog.xml
 
+[files]
+logs/
+
+[ini-template]
+## Format string
+# jetty.customrequestlog.formatString=%a - %u %{dd/MMM/yyyy:HH:mm:ss ZZZ|GMT}t "%r" %s %B "%{Referer}i" "%{User-Agent}i" "%C"
```

Comment:
should this property still be `jetty.customrequestlog` when the rest are just using `jetty.requestlog`

Reply:
thanks lachlan!   But please review rather than comment... so when I fix I can commit!

Reply:
It was a review comment, I didn't want to approve because I don't understand some of the other changes